src/main.rs

```diff
@@ -12,9 +14,15 @@ struct Cli {
 #[derive(Subcommand)]
 enum CliCommand {
     /// Build the message database.
-    Build,
+    Build {
+        /// Path to the backup directory (default: `./data`).
+        path: Option<PathBuf>,
+    },
     /// Serve the content.
-    Serve,
+    Serve {
+        /// Path to the backup directory (default: `./data`).
+        path: Option<PathBuf>,
+    },
```

Comment:
With this new setup, having an explicit separation of subcommands probably makes no sense, so I'd be inclined to remove it (but in a separate PR) as this would also simplify the instructions.